save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/HistoryView.kt

```diff
@@ -41,42 +42,54 @@ external interface HistoryProps : RProps {
  * A table to display execution results for a certain project.
  */
 class HistoryView : RComponent<HistoryProps, RState>() {
-    @Suppress("TOO_LONG_FUNCTION", "ForbiddenComment")
+    @Suppress("TOO_LONG_FUNCTION", "ForbiddenComment", "LongMethod")
     override fun RBuilder.render() {
         child(tableComponent(
             columns = columns {
                 column("index", "#") {
                     td {
-                        +"${it.row.index}"
+                        a(href = "${window.location}/${it.value.id}") {
+                            +"${it.row.index + 1}"
+                        }
                     }
                 }
                 column("status", "Status") {
                     td {
-                        +"${it.value.status}"
+                        a(href = "${window.location}/${it.value.id}") {
```

Comment:
Please create a function for this url, so that if we change it's format, we change it in one place